### ISSUE TITLE
Ability to choose own directory name

### ISSUE THREAD

**Issue by alexharris** [billandalex/banda-installer#1]
It would be nice to able to install into any folder, not just "banda_project".

**Comment by wphogan** [comment]
i don't think this is possible since we're sharing the database on our local machines and all the links in the database are based on the name of the root folder. but we can change the folder name to anything as long as we both change it to the same name. should we change it?

**Comment by alexharris** [comment]
no its fine, maybe the readme should be updated to indicate that it HAS to be called that
